ObjectState/TrackSends.cpp

```diff
@@ -29,6 +29,8 @@
 #include "../reaper/localize.h"
 #include "../SnM/SnM_Dlg.h"
 #include "TrackSends.h"
+#include "Utility/UtilityFunctions.hpp"
+#include <sstream>

```

Comment:
The include could be put in the precompiled header `stdafx.h` (it's only precompiled on Windows).